include/nuttx/mm/map.h

```diff
@@ -73,6 +74,11 @@ struct mm_map_entry_s
 struct mm_map_s
 {
   sq_queue_t mm_map_sq;
+
+#if defined(CONFIG_BUILD_KERNEL) && defined(CONFIG_GRAN)
+  GRAN_HANDLE vpages;
```

Comment:
I know "they" will say this, but add mm_ prefix to keep naming consistent :)

Reply:
but of course! :)

Reply:
done